quickstarts/quickstart-template.md

```diff
@@ -0,0 +1,81 @@
+ 
+
+# Title
+
+:information_source: Before using this template, read the accompanying [quickstart template guide](quickstart-template-guide.md).
+
+## Overview
+
+This quickstart guides you through:
+- [Part 1](#part-1-taskname)
+- [Part 2](#part-2-taskname)
+- [Part n](#part-n-taskname)
```

Comment:
**Suggestion (grammar, blocking)** - Let's make it "part-1-task-name" et. al. instead. If they're generating links inside markdown, that would be the proper format, I believe.

Reply:
Fixed